warehouse/query-core/src/main/java/datawave/query/jexl/lookups/IndexLookup.java

```diff
@@ -93,9 +92,9 @@ protected boolean timedScan(final Iterator<Entry<Key,Value>> iter, final IndexLo
             future.cancel(true);
             if (null != log && log.isTraceEnabled())
                 log.trace("Timed out ");
-            if (fields.size() >= 1) {
-                for (Text fieldTxt : fields) {
-                    String field = fieldTxt.toString();
+            // Only if not doing an unfielded lookup should we mark all fields as having an exceeded threshold
+            if (!unfieldedLookup) {
```

Comment:
If it's NOT an unfielded lookup, and we have 0 fields, do we not want to mark it as KeyThresholdExceeded?

Reply:
I'm good on this per our discussion.

Reply:
no

Reply:
nope